la_funding_analysis/utils/jitter_functions.py

```diff
@@ -0,0 +1,17 @@
+# File: utils/jitter_functions.py
+"""Functions to enable the creation of jitter plots with the jitter in a single direction.
+"""
+
+import numpy as np
+
+
+def rand_jitter(list, deviation):
+    # Function to add small variation to values in a list.
```

Comment:
for all functions with # - make them actual doc strings

Reply:
Good catch, done